orttraining/orttraining/core/framework/torch/custom_function_register.cc

```diff
@@ -105,46 +108,27 @@ void OrtTorchFunctionPool::RegisterInputAliasFunction(const std::string& key,
   RegisterEntry(mutex_, key, obj, input_alias_function_pool_);
 }
 
-static void RegisterEntry(
```

Comment:
Is this duplicate of another `RegisterEntry`?

Reply:
This has a different function signature with another one. After this PR, this overload is not needed, so removed it.